tests/subsys/edac/ibecc/src/ibecc.c

```diff
@@ -322,14 +320,14 @@ void test_main(void)
 		k_oops();
 	}
 #endif
+}
 
-	ztest_test_suite(ibecc,
-			 ztest_unit_test(test_ibecc_initialized),
-			 ztest_unit_test(test_ibecc_api),
-			 ztest_unit_test(test_edac_dummy_api),
-			 ztest_unit_test(test_ibecc_error_inject_api),
-			 ztest_unit_test(test_ibecc_error_inject_test_cor),
-			 ztest_unit_test(test_ibecc_error_inject_test_uc)
-			);
-	ztest_run_test_suite(ibecc);
+ZTEST(ibecc, ibecc_injection)
```

Comment:
Needs to be renames to started with `test_` if I understand right error build log.

Reply:
I will add test_ prefix before ibecc_injection